src/navigation/Breadcrumb/index.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { Icon, IconTypes } from '../../dataDisplay/Icon';
+import Text from '../../dataDisplay/Text';
+import Link from '../../inputs/Link';
+import { ThemeColors } from '../../theme';
+
+const Wrapper = styled.div`
+  padding: 16px 0;
+  box-sizing: border-box;
+  max-height: 54px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`;
+
+const ElementWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  margin: auto 5px;
+`;
+
+const StyledLink = styled(Link)`
+  font-family: ${({ theme }) => theme.fonts.fontFamily};
+  text-transform: uppercase;
+  text-decoration: none;
+  font-weight: 800;
+  cursor: text;
+  margin: auto 5px;
+`;
+
+const StyledText = styled(Text)`
+  font-family: ${({ theme }) => theme.fonts.fontFamily};
```

Comment:
We should fix this. https://material-ui.com/customization/typography/#font-family

Reply:
I was checking just in case. We are using theme from `styled-components`, not from material-ui. Fonts are defined here, so object structure is correct

https://github.com/gnosis/safe-react-components/blob/development/src/theme.ts#L54